azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -214,8 +214,7 @@ def remove_extension_cgroups(self, name):
             Deletes the cgroup for the given extension
             """
             def __impl():
-                cgroups = self._cgroups_api.remove_extension_cgroups(name) # pylint: disable=E1111
-                return cgroups
+                self._cgroups_api.remove_extension_cgroups(name)
```

Comment:
We don't actually have to return here. `_invoke_cgroup_operation` only uses the `operation` parameter's return value as its return value. As we're ignoring said return value, providing one here doesn't do anything for us.

You can see this pattern above, on line 199 of this file, where `_cgroups_api.create_extension_cgroups_root()` also doesn't return anything.